### DISCUSSION TITLE
AuthUser is null in SSR when opening the app url from link

### DISCUSSION THREAD

**Discussion by radim10** [gladly-team/next-firebase-auth#393] [Q&A] [answered]
If I paste the URL it works fine, but when I click for example on link to my app in an email the AuthUser is null. Is this a correct behaviour?

**Comment by kmjennison** [comment, marked as answer]
Yes, and it's determined by your cookie's `SameSite` setting. More info: https://github.com/gladly-team/next-firebase-auth/issues/351#issuecomment-991134374
